BitwardenShared/Core/Platform/Services/Stores/AppSettingsStore.swift

```diff
@@ -281,6 +293,19 @@ protocol AppSettingsStore: AnyObject {
     ///
     func setEvents(_ events: [EventData], userId: String)
 
+    /// Sets a feature flag value in the app's settings store.
+    ///
+    /// This method updates or removes the value for a specified feature flag in the app's settings store.
+    /// If the `value` parameter is `nil`, the feature flag is removed from the store. Otherwise, the flag
+    /// is set to the provided boolean value.
+    ///
+    /// - Parameters:
+    ///   - name: The name of the feature flag to set or remove, represented as a `String`.
+    ///   - value: The boolean value to assign to the feature flag. If `nil`, the feature flag will be removed
+    ///    from the settings store.
+    ///
+    func setFeatureFlag(name: String, value: Bool?)
```

Comment:
🤔 If we're gating using these functions behind an `#if DEBUG`, wouldn't it make sense to also gate these functions and attendant code?

(Other discussion about whether we want to `#if DEBUG` aside)

Reply:
yeah that a great call too, no need to have these functions available unless they are needed.